src/Units/Unit.java

```diff
@@ -201,10 +201,19 @@ public int getPreciseHealth()
   {
     return health;
   }
+  public boolean isHurt()
+  {
+    return health < healthFromHP(model.maxHP);
+  }
   public int getHP()
   {
     return healthToHP(health);
   }
+  /** @return value in range [0-1.0]; represents the unit's current effectiveness */
+  public double getHPRatio()
```

Comment:
I feel like `getPercentHealth` might be more clear; `getHPRatio` sounds like it's comparing two HP values rather than an HP and a max HP.

Reply:
But "percent health" is what `health` already is.
Words are hard.

Reply:
Then is `HPRatio` ever going to be different from `health/100` (barring the case where `maxHP!=100`, which I still see as spurious)?

Reply:
I don't think so? It's just a convenience function, as you can see in the diff.

Reply:
Ah, touché. I'm still not a fan of "ratio" though. Maybe "getHPAsPercent" then? I dunno.